.vsts-ci.yml

```diff
@@ -1,12 +1,15 @@
 variables:
-  Build.Repository.Clean: true
-  _HelixType: build/product
-  _HelixSource: pr/dotnet/arcade-minimalci-sample/$(Build.SourceBranch)
-  _enableTelemetry: true
-  ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
-    OfficialBuildId: $(Build.BuildNumber)
-    DOTNET_SKIP_FIRST_TIME_EXPERIENCE: 1
-    DOTNET_MULTILEVEL_LOOKUP: 0
+- name: Build.Repository.Clean
+  value: true
+- name: _enableTelemetry
```

Comment:
Is this variable still needed? I think this is now a parameter on each Job but don't remember.

Reply:
I am sending it to each job and setting it to true since the job defaults to false.

Reply:
Ah I missed that, this is just a shared variable for your multiple jobs, this is one of the downsides of using the `Job` template, that you duplicate a lot of the parameters, when using the `Jobs` template you can pass multiple jobs down with the same set of parameters.